index.html

```diff
@@ -353,6 +353,35 @@ <h2>
               </tr>
             </tbody>
           </table>
+          <div class="note">
+            <p>Below are few non-normative recommendations when capturing display for 
+              applications that support full screen rendering.
+            </p>
+            <p>
+              For applications that support full screen rendering, like powerpoint for example, 
+              the decision of the display surface that gets captured while in full screen mode is 
+              specific to the user agent implementation. For consistency reasons, 
+              an user agent must ensure that the display surface captured in full screen 
+              mode be unambigously mapped to the application/window where the action was triggered 
+              in the first place. 
+            </p>
+            <p>
+              The choice of <code><a>DisplayCaptureSurfaceType</a></code> and 
+              <code><a data-link-for="MediaTrackConstraintSet">logicalSurface</a></code> may 
+              further influence what gets captured. For example, when an application/window 
+              is triggered to go full screen and the DisplayCaptureSurfaceType of "monitor" 
+              is selected, the choice of the "monitor" display surface selected for
+              capture is solely left to the user agent, especially when there 
+              are mutliple options available.
+            </p>
+            <p>
+             An user agent must also be prepared to handle certain transient behaviors where 
+             the action to capture an application/window in full screen may be reverted, say due
+             to user canceling the action immediately. In these scenarios, the user agent
+             should ensure that the captured display surface is restored to the application/window 
+             originating the action.
```

Comment:
I think that you need to emphasize this in the security considerations.  Possibly with a MUST slapped on it. If the intent of the user is to share powerpoint and a flash of their entire desktop is shown when the presentation ends, that's muy bad.